### ISSUE TITLE
Documentation for Key/Value displayed/consumed is inverted

### ISSUE THREAD

**Issue by westshawn** [grundic/teamcity-web-parameters#2]
Thanks for your work on this great plugin!

I noticed one small issue on https://github.com/grundic/teamcity-web-parameters under the "Web-service" section:
```
Each element in options's array is hash; key will be option's display name, value will be option's value, enabled will determine if option will be enabled.
```
In practice, it seems like the key/value are inverted: the "value" from the hash is displayed in the select list, and the "key" from the hash is being consumed by TeamCity.

**Comment by grundic** [comment]
@westshawn, thank you for you time for creating this issue! I've checked it right now -- everything works just like you said, my bad.
In order to keep compatibility with already running and configured services, I'd better just swap items in documentation.

**Comment by grundic** [comment]
Well, and here is the [bug](https://github.com/grundic/teamcity-web-parameters/blob/master/teamcity-web-parameters-server/src/main/resources/buildServerResources/ru/mail/teamcity/web/parameters/jsp/webParameterControl.jsp#L19-L20) itself

**Comment by grundic** [comment]
@westshawn, the issue is fixed now, you can download version [1.2.0](https://github.com/grundic/teamcity-web-parameters/releases/latest) and check it.
Thank you!

**Comment by westshawn** [comment]
@grundic, 
Thanks for the lightning-fast fix.
It works as expected. 
Thank You!
